app/routes/user.js

```diff
@@ -1,4 +1,5 @@
 import Ember from 'ember';
+import ajax from 'ic-ajax';
```

Comment:
In other cleanup PRs that have been going on, ic-ajax use has been removed in favor of ember-ajax